core/templates/hash_map.h

```diff
@@ -310,6 +310,29 @@ class HashMap {
 		return _lookup_pos(p_key, _pos);
 	}
 
+	void replace(const TKey &p_key, const TKey &new_key) {
```

Comment:
What's the functional difference between this and `replace_key`? Should have a comment above specifying that, also the name is a bit misleading, especially when `replace_key` already exists

Reply:
This is exactly the same at the time II first wrote this part replace_key didn't exist so I had to make it myself and didn't look if it had been implemented since then which is the case I'll update to use replace_key

Reply:
Got it! Didn't realise this was such an old commit :)

Reply:
yeah 😅 the original commit that I salvaged to make this is one year old which predates the `replace_key`. I slowly built the nehancements to the `EditorPropertyDictionary` because it didn't work as one huge PR.